### PR TITLE
Enhancement/219 add kernel vc

### LINKED ISSUE #219: Refactor: kernel configuration menu
Change current kernel config to kernel_vc

### PR DESCRIPTION
**Pull request by ljelen**
closes #219

### CODE REVIEW

**Comment by grzanka** [comment]
image

Rename:  z-> Z

sort symbols accoring to how they appear in periodic table (with increasing Z): H, He, ...

**Comment by nbassler** [comment]
@grzanka a convention is to use z for projectiles, and Z for targets

**Comment by grzanka** [comment]
> @grzanka a convention is to use z for projectiles, and Z for targets

is it TRiP98 convention ?

**Comment by nbassler** [comment]
no, its general, see e.g. Bethe equation.

**Comment by nbassler** [comment]
Ofc it is a bit redundant, the symbol should be enough. Even for antiprotons and pions.

**Comment by grzanka** [comment]
> Ofc it is a bit redundant, the symbol should be enough. Even for antiprotons and pions.

Thanks, so I'd only stay with suggestion of sorting projectile names

File: pytripgui/kernel_vc/kernel_view.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+        kernels = list()
+        for i in range(kernels_count):
+            kernel = self.ui.beamKernel_comboBox.itemData(i, Qt.UserRole)
+            kernels.append(kernel)
+        return kernels
+
+    def replace_kernel_by_index(self, kernel, kernel_index):
+        self.ui.beamKernel_comboBox.setItemData(kernel_index, kernel, Qt.UserRole)
+
+    def setup_all_available_projectile_symbols(self, symbols):
```
Comment (grzanka) [inline pytripgui/kernel_vc/kernel_view.py]: Dictionary of projectiles is not necessarily sorted:
```
>>> 
(pytripgui) grzanka@grzanka-VirtualBox:~/PycharmProjects/pytripgui$ PYTHONPATH=. python 
Python 3.5.6 |Anaconda, Inc.| (default, Aug 26 2018, 21:41:56) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pytrip.tripexecuter import Projectile
>>> Projectile._projectile_defaults
{'H': (1, 1), 'O': (8, 16), 'Ar': (16, 40), 'He': (2, 4), 'Ne': (10, 20), 'Li': (3, 7), 'C': (6, 12)}
>>> for symbol in Projectile._projectile_defaults:
...    print(symbol)
... 
H
O
Ar
He
Ne
Li
C
```

Please add proper sorting here. An example of something ugly:

```
>>> [y[0] for y in sorted(Projectile._projectile_defaults.items(), key=lambda x : x[1][0])]
['H', 'He', 'Li', 'C', 'O', 'Ne', 'Ar']
```

**Comment by grzanka** [comment]
Why have you added  here a commit ece0693 with message `self loading UI`. Is it related with this PR ?

**Pull request merged**